cloud/features/audit-logs.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Audit Logs
+description: Learn about centralized audit logs in Flipt Hybrid Cloud
+mode: "wide"
+---
+
+Flipt Hybrid Cloud provides centralized audit logs to help you track changes to your feature flags and configurations. With audit logs, you can see who made changes, when they were made, and what changes were made. This makes it easy to track changes, troubleshoot issues, and maintain a history of your feature flag configurations.
```

Comment:
Should we throw a note in here that this is only available for the relational backends?

Reply:
yeah probably